src/api/spec/models/obs_factory/distribution_strategy_casp_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
+require 'webmock/rspec'
+
+RSpec.describe ObsFactory::DistributionStrategyCasp do
+  let(:project) { create(:project, name: 'SUSE:SLE-12-SP2:Update:Products:CASP2') }
+  let(:distribution) { ObsFactory::Distribution.new(project) }
+  let(:strategy) { distribution.strategy }
+
+  shared_context 'mock backend call' do
+    let(:staging_project) { create(:project, name: 'SUSE:SLE-12-SP2:Update:Products:CASP2:staging:A') }
+    let(:backend_url) { "#{CONFIG['source_url']}/build/#{staging_project}/_result?view=binarylist&package=CAASP-dvd5-DVD-x86_64&repository=images" }
+    let(:backend_response) do
+      %(<resultlist state='d797d177b6a6a9096ca39b01d40ab600'>
+        <result project="#{staging_project}" repository='images' arch='x86_64' code='unpublished' state='unpublished'>
+          <binarylist package='CAASP-dvd5-DVD-x86_64'>
+            <binary filename='CASP-Build1039.1-Media.iso' size='879214592' mtime='1528427209'/>
+            <binary filename='CASP-Build1039.1-Media.iso.sha256' size='623' mtime='1528427215'/>
+            <binary filename='CASP-Build1039.1-Media.report' size='371629' mtime='1528427188'/>
+            <binary filename='_channel' size='64345' mtime='1528427189'/>
+            <binary filename='_statistics' size='716' mtime='1528427188'/>
+          </binarylist>
+        </result>
+      </resultlist>)
+    end
+  end
+
+  describe '#staging_manager' do
+    it { expect(strategy.staging_manager).to eq('caasp-staging-managers') }
+  end
+
+  describe '#repo_url' do
+    it { expect(strategy.repo_url).to eq('http://download.opensuse.org/distribution/13.2/repo/oss/media.1/build') }
+  end
+
+  describe 'openqa_version' do
+    it { expect(strategy.openqa_version).to eq('1.0') }
```

Comment:
wouldn't expect this either - you checked casp2

Reply:
but as casp2 is no longer in development, it's hard to say. casp3's openqa_version is 3.0, so I wonder why the strategy returned 1.0 for 2

Reply:
@coolo. openqa_version is hardcode with `1.0`

Reply:
Very possible. I'm just saying I wouldn't *expect* current bugs - either fix them or leave them untested

Reply:
So the current `openqa_version` for CASP should be `3.0`, right?